server/routes/caseNotes/checkAnswers/checkAddCaseNoteAnswersPresenter.ts

```diff
@@ -0,0 +1,33 @@
+import { CaseNote } from '../../../models/caseNote'
+import DateUtils from '../../../utils/dateUtils'
+import UserDetails from '../../../models/hmppsAuth/userDetails'
+
+interface CaseNotesSummary {
+  subject: string
+  date: string
+  time: string
+  from: string
+  body: string
+}
+
+export default class CheckAddCaseNoteAnswersPresenter {
```

Comment:
Do you think this class can go in the `/caseNotes/add/` directory (as it's part of that journey) rather than a new `/check-answers` one?

Another nitpicky one, but I also wonder if `AddCaseNoteCheckAnswersPresenter` as a name might be more consistent with other classes in the codebase - I think we have examples of both, but we tend to lean towards having the `CheckAnswersPresenter` after the name of the user journey.

Reply:
Ah thanks for your suggestions. I agree.

I'll create a new directory as:

`/caseNotes/add/check-answers`